### DISCUSSION TITLE
Monitor wraparound

### DISCUSSION THREAD

**Discussion by JamieKM** [MonitorControl/MonitorControl#1019] [Ideas]
I have an idea for a feature I call Wraparound, which comes from an old piece of software of the same name that's no longer supported. The idea is that you could select sides of a monitor that, when enabled, allows the cursor to wrap around to the other edge. For example, in a 2 monitor setup, when your cursor moves to the left edge of the left monitor, instead of stopping, it wraps around to the right edge of the right monitor. Or if your cursor moves to the top of your monitor, instead of stopping, it wraps around  to the bottom of the monitor, basically eliminating monitor edges. On large, 2-monitor systems, it's a godsend. Here's an image of the product's Preferences, indicating wraparound edges on the left edge of the left monitor and right edge of the right monitor:

<img width="607" alt="Screen Shot 2022-04-30 at 7 54 41 AM" src="https://user-images.githubusercontent.com/104679784/166104460-38ec2fa2-afa8-4b49-8d0e-7894c2e626ed.png">

This is a feature I would be happy to fund. Thoughts?

**Comment by waydabber** [comment]
This is a great idea. I'll implement this in BetterDummy probably as it already has awareness of display layout and also has some mouse control logic. Added it as a feature request here: https://github.com/waydabber/BetterDummy/issues/543

**Comment by JamieKM** [comment]
Cool. Let me know how I can support you!

**Comment by JamieKM** [comment]
Couple years later and just wondering if this was ever implemented? Doesn’t appear BetterDummy is still an app? Was its functionality rolled into BetterDisplay?

-Jamie

**Reply by waydabber** [reply]
Yes, the functionality is rolled into BetterDisplay. I added various mouse warp features, including a silent auto-warp feature (https://github.com/waydabber/BetterDisplay/issues/3569), which in some respects is rather similar to what the wraparound should be doing but in a different context. I am still considering the feature and it's in the enhancement list. I don't have an ETA though. Will probably add it eventually.
